feature/experimental/policy/policy_vrf_selection/otg_tests/policy_based_traffic_steering_default_to_nondefault/README.md

```diff
@@ -0,0 +1,311 @@
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+   * ATE2:Port3 and DUT:Port3 from EBGP neighborship between each other over on respective AFI-SAFI connections and advertise the following prefixes
+     * From ATE2:Port2 to DUT:Port2
+       * IPv4Prefix6/24 IPv6Prefix6/64
+       * IPv4Prefix7/24 IPv6Prefix7/64
+       * IPv4Prefix8/24 IPv6Prefix8/64
+       * IPv4Prefix9/24 IPv6Prefix9/64
+       * IPv4Prefix10/24 IPv6Prefix10/64
+     * From DUT:Port2 to ATE2:Port2
+       * All the IBGP learnt prefixes from Prefix1 to Prefix5<br>
+
+
+     * **[Important: Please note]** Following advertisements must happen only during the test one at a time as instructed<br>
+
+* Configure the DUT to honor the TOS byte of destination 
+| Match Condition                           | Outer Header DSCP Set to: |
+|-------------------------------------------|---------------------------|
+| DSCP value BE1, Destination IPv4Prefix6/24"                         | 3                         |
+| DSCP value BE1, Destination IPv6Prefix6/64"                         | 3                         |
+| DSCP value AF1, Destination IPv4Prefix7/24"                         | 11                        |
+| DSCP value AF1, Destination IPv6Prefix7/64"                         | 11                        |
+| DSCP value AF2, Destination IPv4Prefix8/24"                         | 19                        |
+| DSCP value AF2, Destination IPv6Prefix8/24"                         | 19                        |
+| DSCP value AF3, Destination IPv4Prefix9/24"                         | 27                        |
+| DSCP value AF3, Destination IPv6Prefix9/64"                         | 27                        |
+| DSCP value AF4, Destination IPv4Prefix10/24"                       | 35                        |
+| DSCP value AF4, Destination IPv6Prefix10/24"                       | 35                        |
+
+
+
+   * DUT has 2 VRFs, Default and Non-Default. Configure Route leaking from the Default to the Non-default VRF to share reachability between VRFs.<br>
+
+
+### Flows:
+
+Flows from ATE1:Port1 to DUT:Port1
+
+   * Following are destinations.
+       * IPv4Prefix1/24 to IPv4Prefix6/24 [DSCP:BE1]
+       * IPv6Prefix1/64 to IPv6Prefix6/64 [DSCP:BE1]
+       * IPv4Prefix2/24 to IPv4Prefix7/24 [DSCP:AF1]
+       * IPv6Prefix2/64 to IPv6Prefix7/64 [DSCP:AF1]
+       * IPv4Prefix3/24 to IPv4Prefix8/24 [DSCP:AF2]
+       * IPv6Prefix3/64 to IPv6Prefix8/64 [DSCP:AF2]
+       * IPv4Prefix4/24 to IPv4Prefix9/24 [DSCP:AF3]
+       * IPv6Prefix4/64 to IPv6Prefix9/64 [DSCP:AF3]
+       * IPv4Prefix5/24 to IPv4Prefix10/24 [DSCP:AF4]
+       * IPv6Prefix5/64 to IPv6Prefix10/64 [DSCP:AF4] br><br><br>
+
+
+### Following Health checks to be run Pre and Post every sub test
```

Comment:
Same comment as elsewhere -- please can we define common health checks if they are required.